src/App.js

```diff
@@ -1,20 +1,21 @@
-import React, {useEffect, useState} from 'react';
+import React, { useEffect} from 'react';
+import {BrowserRouter as Router, Switch, Route} from "react-router-dom";
+import {Home} from './components/main/Home'
+
```

Comment:
Лучше сортировать импорты. Сначала из либ, потом свои компоненты и тулзы
```suggestion
import React, { useEffect} from 'react';
import {useSelector, useDispatch} from "react-redux";
import {BrowserRouter as Router, Switch, Route} from "react-router-dom";

import {Home} from './components/main/Home'
import SavedCities from "./components/basic/SavedCities";
```